src/authorization/WebAclAuthorizer.ts

```diff
@@ -192,6 +198,17 @@ export class WebAclAuthorizer extends Authorizer {
       if (acl.countQuads(auth, ACL.agent, agent.webId, null) !== 0) {
         return true;
       }
+
+      // Check if authenticated access is allowed
```

Comment:
@joachimvh To do: move the other checks to access checkers.

Reply:
Fixed